mscore/menus.cpp

```diff
@@ -1308,6 +1308,65 @@ void MuseScore::setBasicPalette()
       }
 
 //---------------------------------------------------------
+//   genCreateMenu
+//---------------------------------------------------------
+
+QMenu* MuseScore::genCreateMenu(QWidget* parent)
```

Comment:
I'm not sure why this shows up in the diff, I didn't add a new function per se, but a "sub-menu", if you will.